### PR TITLE
Add Yale Access Bluetooth integration

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add Yale Access Bluetooth integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76182
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3570
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/yalexs_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- YRD216
+- YRL216
+- YRD226
+- YRL226
+- YRD256
+
+## Obtaining the offline key
+
+The offline key and slot number are required to operate the lock. These credentials reside in the Yale Access app storage on iOS or Android devices with owner access to the lock.
```
Comment (bdraco) [inline source/_integrations/yalexs_ble.markdown:48]: The next steps will pull the keys from the cloud without all this but thats not in the first PR.

**Comment by bdraco** [comment]
Might have to make a note that it can take a while to lock/unlock if we don't get https://github.com/hbldh/bleak/pull/923 done in time for 2022.9

File: source/_integrations/yalexs_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+- YRD216
+- YRL216
+- YRD226
+- YRL226
+- YRD256
+
+## Push updates
+
+Some locks only sends push updates when they have an active HomeKit pairing. If your lock is not getting push updates, make sure its paired with a HomeKit controller. The lock cannot be paired via HomeKit Controller and the Yale Access Bluetooth integration on the same Home Assistant instance as they will both try to access the lock at the same time and fail.
```
Comment (bdraco) [inline source/_integrations/yalexs_ble.markdown]:
```suggestion
Some locks only send push updates when they have an active HomeKit pairing. If your lock is not sending push updates, ensure it's paired with a HomeKit using an iOS device or the HomeKit controller integration. The lock cannot be paired via HomeKit Controller and the Yale Access Bluetooth integration on the same Home Assistant instance as they will both try to access the lock simultaneously and fail.

Alternatively, call the `homeassistant.update_entity` service to force the integration to update the lock state.
```

File: source/_integrations/yalexs_ble.markdown [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Supported devices
+
+Devices must have the Yale Access module installed.
+
+- YRD216
+- YRL216
+- YRD226
+- YRL226
+- YRD256
```
Comment (bdraco) [inline source/_integrations/yalexs_ble.markdown:32]: I just got the conexis l1 working yesterday so I need to add that to the list

**Review by frenck — APPROVED** [review]
Thanks, @bdraco

**Comment by bdraco** [comment]
Thanks

**Pull request merged**